test/test_models.py

```diff
@@ -38,13 +38,17 @@ def get_available_video_models():
 # models that are in torch hub, as well as r3d_18. we tried testing all models
 # but the test was too slow. not included are detection models, because
 # they are not yet supported in JIT.
+
+# Temp disabled modules: disabled until https://github.com/pytorch/pytorch/pull/28988 lands
+# so that we do not have to introduce BC changes.
+
 script_test_models = [
-    "deeplabv3_resnet101",
+    # "deeplabv3_resnet101", temp disabled
```

Comment:
If this is all going in before a release it's better to just wait until pytorch#28988 lands then land this PR so these tests never have to be disabled

Reply:
AFAIK There's a minor release of torchvision coming up, this PR is for that release

Reply:
The minor release of torchvision didn't contain the BC-breaking commits, so I think we can wait until pytorch#28988 lands to merge this PR, thoughts?

Reply:
Okay cool, in that case yes we can wait.

Reply:
@eellison https://github.com/pytorch/pytorch/pull/28988 has landed, can you re-enable this test?